src/lib/import_export/binary/binary_writer.cpp

```diff
@@ -344,6 +349,9 @@ uint32_t BinaryWriter::_compressed_vector_width(const AbstractEncodedSegment& ab
       case CompressedVectorType::FixedSize1ByteAligned:
         vector_width = 1u;
         break;
+      case CompressedVectorType::FixedSizeBitAligned:
+        vector_width = 0u;
```

Comment:
I'm aware that I'm abusing this `vector_width` but wanted to check what to before renaming all of this to `vector_compression_id` ?

Reply:
used the `CompressedVectorType` enum values as ids and renamed all notions of this `vector_width` that was actually used as id to `vector_compression_id`. In order to maintain compatibility with old binary exports, i changed the order of the `CompressedVectorType` enum values to match the exported indices.

Reply:
There is no need to keep compatibility with old  binaries.

Reply:
Okay, I'll just leave it as it is, doesn't hurt